submissions/sprint_c/test_helpers.py

```diff
@@ -0,0 +1,198 @@
+import unittest
+import os
+import shutil
+from flask import json
+from helpers import FileManager, Vocabulary, EpithetGenerator
+
+
+class HelperTests(unittest.TestCase):
+
+    path = './temp_dir/data.json'
+    sad_path = './fake_dir/data.csv'
+
+    def setUp(self):
+        data = {
+            "Column1": "bar",
+            "Column2": "foo",
+            "Column3": "baz"
+        }
+
+        if os.path.isdir('temp_dir'):
+            shutil.rmtree('temp_dir')
+            os.mkdir('./temp_dir')
+            with open('./temp_dir/data.json', 'w') as test_file:
+                json.dump(data, test_file)
+        else:
+            os.mkdir('./temp_dir')
+            with open('./temp_dir/data.json', 'w') as test_file:
+                json.dump(data, test_file)
+
+    def tearDown(self):
+        shutil.rmtree('./temp_dir')
+
+    def test_get_extension(self):
+        self.assertEqual(FileManager.get_extension(self.path), 'json')
+
+    def test_sad_path_get_extension(self):
+        try:
+            self.assertEqual(FileManager.get_extension(self.sad_path), 'json')
+        except Exception as ex:
```

Comment:
Don't use try/except exception handling in tests. Use assert raises instead. 

https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions